app/cdap/components/PipelineConfigurations/TabConfig.js

```diff
@@ -60,6 +61,13 @@ const TabConfig = {
       contentClassName: 'pipeline-configurations-body',
       paneClassName: 'configuration-content',
     },
+    {
```

Comment:
Might have to add a conditional here. We don't want to show this setting on realtime pipeline